### PR TITLE
Refine metrics CI invocations

### PR DESCRIPTION
**Pull request by grahamwhaley**
Refine our metrics CI invocations somewhat:
- stop running tests that we don't currently check the results for
- refine the iperf3 test run to only test what we do check
    - which involved adding a commandline to the iperf3 test so we could tweak it

### CODE REVIEW

**Comment by grahamwhaley** [comment]
A quick note on the README update - yes, the two spaces added to the end of a line in the bullet list are deliberate - they force a line break, and hence a new paragraph, within that bullet.
Dropping the WIP now...

File: .ci/run_metrics_ci.sh [hunk 1]
```diff
@@ -125,18 +125,8 @@ pushd "$CURRENTDIR/../metrics"
 	# ops/second
 	bash network/network-nginx-ab-benchmark.sh
 
-	# ping latency
-	bash network/network-latency.sh
```
Comment (jodh-intel) [inline .ci/run_metrics_ci.sh]: I see that we still run these tests in `metrics/run_all_metrics.sh`, but I'm now wondering if we shouldn't just have a single parameterised script to handle all metrics tests?

Comment (grahamwhaley) [reply .ci/run_metrics_ci.sh]: as in a single script where we can say 'script --all' or 'script --kpis' for instance, to remove the duplication between the two scripts?

We could do that. In my mind we'd re-factor the run_all_metrics (well, and change the name). We'd have to add the KSM handling and some other trickery from the run_metrics_ci script as well (like the results storing options as well).

How about I open an Issue to note we could do with a refactor of the scripts?

Comment (jodh-intel) [reply .ci/run_metrics_ci.sh]: Thanks - Sounds good to me.

**Comment by devimc** [comment]
lgtm

File: .ci/run_metrics_ci.sh (hunk 1 above)
Comment (jodh-intel) [reply .ci/run_metrics_ci.sh]: See #912.

**Comment by jodh-intel** [comment]
lgtm

@klynnrif - could you take a look please?

File: metrics/network/README.md [hunk 2]
```diff
@@ -35,7 +35,9 @@ and packet-per-second throughput using iperf3 on single threaded connections.
 The bandwidth test shows the speed of the data transfer. On the other hand,
 the jitter test measures the variation in the delay of received packets.
 The packet-per-second tests show the maximum number of (smallest sized) packets
-we can get through the transports.
+we can get through the transports.
```
Comment (klynnrif) [inline metrics/network/README.md]: allowed through the transports.

File: metrics/network/README.md [hunk 3]
```diff
@@ -35,7 +35,9 @@ and packet-per-second throughput using iperf3 on single threaded connections.
 The bandwidth test shows the speed of the data transfer. On the other hand,
 the jitter test measures the variation in the delay of received packets.
 The packet-per-second tests show the maximum number of (smallest sized) packets
-we can get through the transports.
+we can get through the transports.  
+Command line options are used to choose which tests to run, for how long, and
```
Comment (klynnrif) [inline metrics/network/README.md]: Command-line options choose what tests to run, for how long, and...

**Comment by grahamwhaley** [comment]
@klynnrif updated, thanks.

**Comment by grahamwhaley** [comment]
hmm, looks like something went wonky with my rebase - let me fix and push that again..

**Comment by grahamwhaley** [comment]
fixed

**Pull request merged**